static/app/components/replays/replayController.tsx

```diff
@@ -167,6 +167,10 @@ const ButtonGrid = styled('div')`
 const TimelineRange = styled(RangeSlider)`
   flex-grow: 1;
   margin-top: ${space(1)};
+  input {
+    padding: 16px 0 16px;
+    margin: -8px 0 0px;
```

Comment:
can we use `space()` for these values?

And also include `cursor: pointer;` so folks know both the existing purple line and the space around it are click/grabbable.

Reply:
Also check in FF. it looks much taller there. I think we need to reset the `height`